### PR TITLE
Add cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability to bitcoin-review-podcast

### PR DESCRIPTION
**Pull request by satstacker21**
This PR adds [Cashu & Fedimint: ecash, Bitcoin's layer 3 & scalability](https://www.youtube.com/watch?v=-nzVYVcdk9M) transcript to the bitcoin-review-podcast directory.

### CODE REVIEW

File: bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md [hunk 1]
```diff
@@ -0,0 +1,2499 @@
...
+Rijndael: 01:51:01
+
+All right.
+Now that Matt's gone, let's get into the real talk about privacy. I'm just kidding.
+
+NVK: 01:51:06
+
+Well, you have nothing to hide, so.
+
+## Scriptable ecash01:59:36 Nostr + ecash
```
Comment (kouloumos) [inline bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md:2072]: The AI transcriptions and the final construction of this document are automated processed that we always try to refine and make them better. Here you can see an example of a mistake in the YouTube chapters of the original source (you can confirm it by looking at the description of the video) where a chapter title was merged with the previous one. You've got the timestamp (01:59:36), so it's easy to place it in the correct position.

File: bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md [hunk 2]
```diff
@@ -0,0 +1,2499 @@
...
+
+## ecash integration and intercompatability
+
+
+Calle: 02:06:26
+
+But can I come back to Eric, what you said about using eCash directly instead of Lightning invoices?
+I have to think more about this.
+I think I agree if you're talking about user-facing Lightning wallets, right?
+So in the best case, a cashu wallet, also a FediMint wallet, right?
```
Comment (kouloumos) [inline bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md:2346]: While the use of ', right?' at the end of a sentence makes sense when a speaker is genuinely seeking confirmation, in many instances, such as this example, it appears as a filler. In this transcript, there are 76 occurrences of ', right?', and the majority of them can be safely removed for a more streamlined transcr

File: bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md [hunk 3]
```diff
@@ -0,0 +1,2499 @@
...
+NVK: 00:06:16
+
+How convenient.
+
+Calle: 00:06:17
+
+Yeah, I think so.
+If things would have gone a little bit more different back in the 90s, I think the default online payment experience today could have been, without Bitcoin and everything before, could have been perfectly private e-cash on the fiat system.
+That is technically possible.
+I know regulatory problems then started with, you know, everything that's happened later.
```
Comment (kouloumos) [inline bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md:217]: Observing the transcript, it's clear that the phrase ', you know,' is frequently employed as a filler, with 93 occurrences. To improve overall clarity and coherence, consider removing it where possible.

**Review by kouloumos — COMMENTED** [review]
My feedback for this aligns with my observations in #378, and I apologize for not addressing it more promptly. Typically, we aim for speedy evaluations of submissions, which can aid in your understanding of our expectations for future reviews.

Given the frequent back-and-forth between participants in this kind of transcripts, it's essential to eliminate stammering, filler words ('like,' 'you know,' ', right?'), and unnecessary non-verbal communication. I've left a few comments inline with some examples. This refinement ensures a coherent and easily readable transcript. This aspect requires a bit more attention before we're ready to merge.

As mentioned in your other review, we are actively working on integrating the review guidelines into the website to ensure a shared standard of transcript quality.

Thank you for your understanding and ongoing contributions!

**Comment by satstacker21** [comment]
Thank you for these corrections.

**Comment by kouloumos** [comment]
Amazing editing! Tackling a transcript that's over 2 hours long is no small feat, and your meticulous attention to detail has significantly enhanced its readability and overall quality.

You are among the few brave contributors who have stepped up to edit such long transcripts, and your dedication is truly commendable. I hope that in the process of this detailed review, you've gained some valuable insights and knowledge.

Don't forget, we have a Discord server [here](https://discord.gg/76wCxgeyvM) where you can connect with other contributors, share experiences/feedback, ask questions and get updates on new transcripts available for review.

We're looking forward to seeing more of your excellent work in the future, @satstacker21. Keep up the fantastic effort!

**Pull request merged**